Automattic/jetpack
# Add the PopupMonitor lib

This updates our usage of PopupMonitor to use a package exported from Calypso.

#### Changes proposed in this Pull Request:

* Use the Calypso package
* Update the API call to get the connect URL

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Update to existing feature

#### Testing instructions:






* Checkout this branch
* Run `yarn` to install the new package
* Add an Instagram Gallery to your post
* Click the connect button
* Check that the popup still opens

* Also navigate to your traffic settings: `/wp-admin/admin.php?page=jetpack#/traffic`
* Click on Verify With Google
* Check that a dialog pops up

#### Proposed changelog entry for your changes:

* Use PopupMonitor from Calypso
